src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -342,3 +344,36 @@ fun wrapWithScrollPane(panel: JPanel): JScrollPane {
     )
     return scrollPane
 }
+
+// "stolen" from https://stackoverflow.com/a/12053940/7577274
```

Comment:
Please have a look at `StringEscapeUtils.escapeHtml`. That should do the job and is part of our dependencies anyway.

Reply:
https://howtodoinjava.com/java/string/escape-html-encode-string/

Reply:
Here we need not only escape html but also replace `\n` and convert html link into real html link (`<href>`)
But good point to delegate that escaping to the library!